spec/builder/custom_builder_spec.rb

```diff
@@ -73,10 +73,21 @@ def to_html
       # See: https://github.com/justinfrench/formtastic/issues/657
       it "should not conflict with navigasmic" do
         allow_any_instance_of(self.class).to receive(:builder).and_return('navigasmic')
-        
+
         expect { semantic_form_for(@new_post) { |f| } }.not_to raise_error
       end
 
+      it "should use the custom builder's skipped_columns config for inputs" do
+        MyCustomFormBuilder.skipped_columns = [:title, :created_at]
```

Comment:
This can affect other tests because it is done on the class level. It would be better to create an anonymous class just in this test case.